.github/labeler.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Labeler
+
+libraries:
+  - lib/**/*
+  - include/grass/*
+  - include/grass/**/*
+  - python/**/*
+
+module:
+  - db/**/*
+  - display/**/*
+  - general/**/*
+  - imagery/**/*
+  - misc/**/*
+  - ps/**/*
+  - raster/**/*
+  - raster3d/**/*
+  - scripts/**/*
+  - temporal/**/*
+  - vector/**/*
+
+# helperscripts:
+#  - utils/*
+#  - utils/**/*
```

Comment:
Remove commented-out code, but you can include comment on what is included and what is not.